.gitignore

```diff
@@ -69,6 +69,7 @@ website/mfrcache/
 website/mfrtemp/
 website/osfstoragecache/
 website/gpg/
+website/static/sitemaps/
```

Comment:
If this is done, another PR will need to update the routes on develop, and check with @mattclark if that changes how he is handling the sitemap routes on staging (I recall he did something a little differently, but that could be false). Happy to make the develop changes, if this change is desirable.